tensorflow/contrib/cmake/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//tensorflow:tensorflow.bzl", "py_test")
+
+py_test(
+    name = "python_modules_test",
+    size = "small",
+    srcs = ["python_modules_test.py"],
+    data = [
+        "//tensorflow:all_contrib_files",
```

Comment:
This includes more than what we want in the pip package.
You may want to check this rule instead:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/BUILD#L11

Reply:
So, would `//tensorflow/contrib:all_files` work as `deps` ?

Reply:
This dependency isn't included in the PIP package. In fact, `pip_smoke_test.py` complained about it not being included, which is why I had to blacklist this.

Reply:
No, what I meant is, why do you need this here?
Our "golden" source for our pip package files is the bazel build rule. While this "all_opensource_files" rule includes everything, many of these may or may not be in the pip package. While for the simpler case of checking if the cmake entries are valid, this can be useful, checking if the cmake build is in parity with the bazel build is a more interesting check, IMHO.

Reply:
Comparing CMake to Bazel wasn't my intention. The tests were originally supposed to check validity. Now I am also linting packages that are expected to be included. Analyzing the PIP package was also not planned. All of this can of course be added later.